packages/release/systemd-pcrphase-configured.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=TPM PCR Barrier (Configured)
+After=configured.target tpm2.target
+Before=activate-multi-user.service systemd-pcrphase-multi-user.service
+ConditionSecurity=tpm2
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/bin/rottweiler measure pcrphase configured
```

Comment:
Not sure where else to leave this comment, but it's really a comment on a commit message note:

> Since the ephemeral storage key is also sealed to the phase PCR,
it means that it can only ever be initialized during that phase. If
the initialization step is moved from "bootstrap container" to
"bootstrap command", then the PCR won't (yet) have reached the state
required to unlock it and the key couldn't be unsealed.

Might be a good idea to document this somewhere (website?) It's a sensible restriction but I could see it being a bit confusing to stumble into.